### PR TITLE
[DEV-11014] - moved session interactive to access command and made it simpler

### PR DESCRIPTION
**Pull request by tomerstein**

### CODE REVIEW

File: pkg/services/sessions.go [hunk 1]
```diff
@@ -26,3 +43,82 @@ func ListAccessSessions(ctx context.Context, client *aponoapi.AponoClient, integ
...
+		var outputBytes []byte
+		outputBytes, err = json.Marshal(accessDetails.Json)
+		if err != nil {
+			return "", err
+		}
+		output = string(outputBytes)
+	}
+
+	return output, nil
+}
```
Comment (tomerstein) [inline pkg/services/sessions.go:113]: Moved from `commands/access/use.go` file for reuse purpose, NO LOGIC CHANGE, only changed error messages

File: pkg/utils/strings.go [hunk 2]
```diff
@@ -6,3 +6,12 @@ func IsValidUUID(u string) bool {
 	_, err := uuid.Parse(u)
 	return err == nil
 }
+
+func Contains(array []string, str string) bool {
```
Comment (naorgu) [inline pkg/utils/strings.go:10]: what about using the super intelligent `slices.contains` function instead ?

**Review by naorgu — APPROVED** [review]

File: pkg/commands/access/actions/access.go [hunk 3]
```diff
@@ -12,7 +19,77 @@ func Access() *cobra.Command {
...
+		return err
+	}
+
+	if len(session.ConnectionMethods) == 0 {
+		return fmt.Errorf("no connection methods found for session %s", session.Id)
+	}
+
+	if len(session.ConnectionMethods) == 1 {
+		err = printSessionInstructions(cmd, client, session)
+		return err
```
[the remark this answers is not in the document]
Comment (tomerstein) [reply pkg/commands/access/actions/access.go:49]: ye lets change it then

File: pkg/utils/strings.go (hunk 2 above)
Comment (tomerstein) [reply pkg/utils/strings.go:10]: I did it but this package no in go version `1.20` which is the version used by this repo.

**Pull request merged**